src/pg_cron.c

```diff
@@ -530,8 +618,13 @@ PgCronLauncherMain(Datum arg)
 
 		AcceptInvalidationMessages();
 
-		if (!CronJobCacheValid)
+		if (!CronJobCacheAndEreportValid)
 		{
+			/* set the desired log_min_messages */
+			ProcessConfigFile(PGC_SIGHUP);
+			SetConfigOption("log_min_messages", cron_error_severity(CronLogMinMessages),
```

Comment:
Looking into SetConfigOption a little further, it looks like it keeps adding to an ever-growing stack of values in order to correctly support (subtransaction) abort. We may need ResetAllOptions or another way to do this.

Reply:
Looked at it and it looks like we are **not** stacking as we end up with ```makeDefault``` being true and so ``` push_old_value()```not being called.
Updating the PR with a new ```CronReloadConfig``` flag.